File: src/main/java/uk/co/halfninja/randomnames/Randomness.java
package uk.co.halfninja.randomnames;

import java.util.List;
import java.util.Random;

class Randomness {
	public static final Random random = new Random();
	public static final String random(List<String> strings) {
		return strings.get(random.nextInt(strings.size()));
	}
}

File: src/main/kotlin/uk/co/halfninja/randomnames/Randomness.kt
package uk.co.halfninja.randomnames

import java.util.Random

object Randomness {
    fun random(): Random = Random()
    fun random(seed: Long): Random = Random(seed)
    fun random(strings: List<String>, random: Random): String = strings[random.nextInt(strings.size)]
}
